src/org/opensolaris/opengrok/util/Executor.java

```diff
@@ -136,6 +136,21 @@ public int exec(boolean reportExceptions) {
         return ret;
     }
 
+    /**
+     * Close all the 3 streams of a process.
+     * @param process
```

Comment:
Add description to the parameter or delete it. This way it says nothing and just takes one line of code...

Reply:
added